docs/rfds/acp-agent-registry.mdx

```diff
@@ -0,0 +1,125 @@
+| `homepage`       | URL for docs/marketing.                                                                                                                                                                                                                                                                                                                                            |
+| `repository`     | Source repository URL.                                                                                                                                                                                                                                                                                                                                             |
+| `authors`        | Array of author/organization names (mirrors `authors` in the TOML example).                                                                                                                                                                                                                                                                                        |
+| `license`        | SPDX identifier or `"proprietary"`.                                                                                                                                                                                                                                                                                                                                |
+| `capabilities`   | Array of ACP method names implemented (e.g. `["terminal/new","files/read"]`).                                                                                                                                                                                                                                                                                      |
+| `auth`           | Array of auth options for authentication. This is the trickiest part of the schema.                                                                                                                                                                                                                                                                                |
+| `distribution`   | _Optional._ Object mapping target platforms to download/execution info. Each target key follows `<os>-<arch>` format (e.g., `darwin-aarch64`, `linux-x86_64`, `windows-x86_64`). Each target specifies `archive` (download URL), `cmd` (executable path), optional `args` (array of command-line arguments), and optional `env` (object of environment variables). |
```

Comment:
Why optional? I assume this is largely the point?
Though maybe we offer a way to discriminate between different distribution types in the future?